src/client/options/mod.rs

```diff
@@ -500,8 +501,11 @@ pub struct ClientOptions {
     /// supported and is considered undefined behaviour. To run any command with a different API
     /// version or without declaring one, create a separate client that declares the
     /// appropriate API version.
-    #[builder(default, setter(skip))]
```

Comment:
Can we also update the builder for `ConnectionPoolOptions` to not skip this field? Alternatively we could just remove the builder API for `ConnectionPoolOptions` as it isn't a public-facing type and can be constructed from a `ClientOptions`, which does have a builder.

Reply:
Removed the builder API.